app/Templates/partials/nav.html.twig

```diff
@@ -22,31 +22,31 @@
                <i class='fa fa-envelope'></i>
                Contact</a>
            {% endif %}
-           {% if enable_connection and enable_connection_tab %}
+           {% if enable_profile == false %}
             <a class="nav-item nav-link {{ class == 'connection' ? 'active'}}"
                 href="{{path_for('connection')}}">
                <i class='fa fa-user-circle'></i>
                Connection</a>
            {% endif %}
-           {% if enable_register and enable_register_tab %}
+           {% if enable_profile == false %}
```

Comment:
Does this mean register link is no longer available if profiles are enabled?

Reply:
Yes because yes you are connected you don't need to register an other account

Reply:
Yes, OK, but looking at the names, this is not explicit (see previous comment)